[2022-06-16 10:33:16,124] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: analytics.getdata scheduled__2022-06-15T00:00:00+00:00 [queued]>
[2022-06-16 10:33:16,163] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: analytics.getdata scheduled__2022-06-15T00:00:00+00:00 [queued]>
[2022-06-16 10:33:16,166] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 10:33:16,169] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-16 10:33:16,171] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 10:33:16,248] {taskinstance.py:1377} INFO - Executing <Task(MySqlOperator): getdata> on 2022-06-15 00:00:00+00:00
[2022-06-16 10:33:16,255] {standard_task_runner.py:52} INFO - Started process 3658 to run task
[2022-06-16 10:33:16,270] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'analytics', 'getdata', 'scheduled__2022-06-15T00:00:00+00:00', '--job-id', '466', '--raw', '--subdir', 'DAGS_FOLDER/testing_etl.py', '--cfg-path', '/tmp/tmp6vfnnrpr', '--error-file', '/tmp/tmpmtn82xud']
[2022-06-16 10:33:16,278] {standard_task_runner.py:80} INFO - Job 466: Subtask getdata
[2022-06-16 10:33:16,575] {task_command.py:370} INFO - Running <TaskInstance: analytics.getdata scheduled__2022-06-15T00:00:00+00:00 [running]> on host fe209149cfc1
[2022-06-16 10:33:17,379] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=analytics
AIRFLOW_CTX_TASK_ID=getdata
AIRFLOW_CTX_EXECUTION_DATE=2022-06-15T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-15T00:00:00+00:00
[2022-06-16 10:33:17,392] {mysql.py:82} INFO - Executing: SELECT * from Customer_T;
[2022-06-16 10:33:17,482] {base.py:68} INFO - Using connection ID 'mysql_con' for task execution.
[2022-06-16 10:33:17,539] {dbapi.py:213} INFO - Running statement: SELECT * from Customer_T;, parameters: None
[2022-06-16 10:33:17,555] {dbapi.py:221} INFO - Rows affected: 19
[2022-06-16 10:33:17,695] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=analytics, task_id=getdata, execution_date=20220615T000000, start_date=20220616T103316, end_date=20220616T103317
[2022-06-16 10:33:17,808] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-16 10:33:17,915] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
